javadoc-parser/src/main/java/com/github/chhorz/javadoc/JavaDocParserBuilder.java

```diff
@@ -29,49 +30,74 @@
  */
 public class JavaDocParserBuilder {
 
-	private static final String BASE_INLINE_PATTERN = "\\{@%s ([^\\{\\}]+)\\}([\\s.,:;-])?";
-	private static final String INLINE_SUMMARY_PATTERN = String.format(BASE_INLINE_PATTERN, "summary");
-	private static final String INLINE_CODE_PATTERN = String.format(BASE_INLINE_PATTERN, "code");
-	private static final String INLINE_LINK_PATTERN = String.format(BASE_INLINE_PATTERN, "link");
-	private static final String INLINE_LINKPLAIN_PATTERN = String.format(BASE_INLINE_PATTERN, "linkplain");
-	private static final String INLINE_LITERAL_PATTERN = String.format(BASE_INLINE_PATTERN, "literal");
-	private static final String INLINE_VALUE_PATTERN = String.format(BASE_INLINE_PATTERN, "value");
-
 	private final JavaDocParser javaDocParser;
 
-	private JavaDocParserBuilder() {
+	private JavaDocParserBuilder(BlockTag... tags) {
 		this.javaDocParser = new JavaDocParser();
-
-		Stream.of(new AuthorTag(),
-						new CategoryTag(),
-						new DeprecatedTag(),
-						new ExceptionTag(),
-						new ParamTag(),
-						new ReturnTag(),
-						new SeeTag(),
-						new SinceTag(),
-						new ThrowsTag(),
-						new VersionTag())
-				.forEach(javaDocParser::addTag);
+		Arrays.asList(tags).forEach(javaDocParser::addTag);
 	}
 
 	/**
-	 * Create a new builder instance with common JavaDoc tags.
+	 * Create a new builder instance without any javadoc tags.
 	 *
 	 * @return a new builder instance
 	 */
-	public static JavaDocParserBuilder withBasicTags() {
+	public static JavaDocParserBuilder withoutTags(){
 		return new JavaDocParserBuilder();
 	}
 
 	/**
-	 * Adds a custom JavaDoc tag to the parser instance.
+	 * Adds all tags from the official <a href="https://docs.oracle.com/en/java/javase/18/docs/specs/javadoc/doc-comment-spec.html">Oracle documentation</a>.
+	 *
+	 * @return a new builder instance
+	 */
+	public static JavaDocParserBuilder withStandardJavadocTags(){
```

Comment:
Good idea to divide javadoc and kdoc tags, but maybe need add also universal parser? For example my case: I don't know which language user use and all what I need in annotation processor - just get maximum information from code to build swagger file automatically. That's why I want to run parser parser with all known tags for all languages. 

The second problem - what about nt official tags like a summary?

I think we need to add one more method:
```java
public static JavaDocParserBuilder withAllKnownTags(){
}
```

Reply:
A new method with all tags has been added. The Summary-Tag is an official one ([Documentation](https://docs.oracle.com/en/java/javase/18/docs/specs/javadoc/doc-comment-spec.html#summary)). The `{@summary}` is an inline tag and those tags are only used for replacements (markdown or asciidoctor). 

For the summary there is a special behaviour in the `JavaDoc` class. If an `{@summary ...}` tag is present the content of this tag will be used - otherwise the first sentence of the Javadoc text will be returned.

Reply:
Hm... interesting. I didn't see this documentation. Oh, summary tag was added in JDK 10... Ok, I see i missed some tags in issue #25. Ok, need t oadd some more tags:
```
@hidden
@index
@provides
@systemProperty
```